Maquetacion/scss/tech.scss

```diff
@@ -0,0 +1,35 @@
+.tech {
+  background-color: $green;
+  border-radius: 5px;
+  box-shadow: -10px -10px $white, -10px -10px 0px 1px $green;
+  display: flex;
+  flex-direction: column;
+  height: 320px;
+  margin: 5px 150px;
+  text-align: center;
+  width: 220px;
+
+  .logo {
+    margin: 15px auto;
```

Comment:
you can avoid this by using a flex property

Reply:
the css 'align-items: center;' is used